### PR TITLE
fdisk: add page

### PR DESCRIPTION
**Pull request by avalanchy**
I think the most common usage, listing available devices.

### CODE REVIEW

File: pages/linux/fdisk.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# fdisk
+
+> Manipulate disk partition table.
+
+- View all available partitions:
```
Comment (agnivade) [inline pages/linux/fdisk.md]: Add the option to change partition table - `fdisk {{/dev/sda}}` first.

Comment (avalanchy) [reply pages/linux/fdisk.md]: Ok fine, I added that but as a second option. Example of editing before listing does not make any sense.

Comment (agnivade) [reply pages/linux/fdisk.md]: Agree.

**Comment by avalanchy** [comment]
:tada:

**Pull request merged**